components/FavMarkers.js

```diff
@@ -0,0 +1,18 @@
+import React from 'react'
+import { Marker } from 'react-native-maps'
```

Comment:
Nice that you added this as its own component. Also that you found out how to use it!